### ISSUE TITLE
Mutation Spectrum not working for brca_tcga

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#2450]
http://www.cbioportal.org/index.do?session_id=591cb29e498e5df2e2917984

**Comment by adamabeshouse** [comment]
I can't reproduce this on local `release-1.7.0`, hopefully it is fixed by the upcoming release.

**Comment by adamabeshouse** [comment]
can't reproduce in beta portal

**Comment by adamabeshouse** [comment]
this is bizarre..... it's not working in production. so it must be a memory issue

**Comment by jjgao** [comment]
@adamabeshouse have you tried to link to the production database and debug? Wondering if there is any data issue.

**Comment by adamabeshouse** [comment]
This is a very strange bug.

So on local, I am reproducing the bug when connecting to the production database. It doesn't seem to be present in any other study. It also seems to be present no matter how many samples from `brca_tcga` I query - even a small subset has the issue. And larger studies (e.g. `brca_metabric`) don't have the problem. So it doesn't seem to be a memory issue on the query side.

So I went in and debugged. Everything proceeds as expected all the way to the database query `getSNPCounts`, the result is coming back empty. Strangely, when I connect to production in Sequel Pro and make the same exact query that it would be making, I get results, it is not empty, so there's no data issue. 

And there's also no exception in Java, e.g. out of memory too many rows. And to double-check that there shouldn't be any memory issue on the MySQL side, I confirmed that the same query for `msk_impact_2017` would be about 4 times as many rows as the result for `brca_tcga`, and so the fact that it's not failing either in the browser or in Sequel Pro means it's not a memory issue at all.

As far as I can tell the only layer where something can still go wrong here is in the formulation of the SQL query in MyBatis. I'm not sure how to figure that out - I've been trying for a bit to find a way to log mysql queries without messing with any production settings. Maybe @sheridancbio or @n1zea144 could help with this?
